L1Trigger/TrackFindingTracklet/python/Producer_cfi.py

```diff
@@ -15,6 +15,8 @@
   BranchLostStubs      = cms.string  ( "StubLost"      ),                                  #
   BranchLostTracks     = cms.string  ( "TrackLost"     ),                                  #
   CheckHistory         = cms.bool    ( False ),                                            # checks if input sample production is configured as current process
-  EnableTruncation     = cms.bool    ( True  ),                                            # enable emulation of truncation, lost stubs are filled in BranchLost
+  EnableTruncation     = cms.bool    ( True  ),                                            # enable emulation of truncation for IRin, TBout, KF, KFin, lost stubs are filled in BranchLost
```

Comment:
Comment for EnableTruncation should not refer to IRin.

Reply:
done